subsys/emul/sensor/icm42688/include/zephyr/emul/sensor/icm42688/icm42688_emul.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_EMUL_SENSOR_ICM42688_INCLUDE_ZEPHYR_EMUL_SENSOR_ICM42688_ICM42688_EMUL_H
+#define ZEPHYR_SUBSYS_EMUL_SENSOR_ICM42688_INCLUDE_ZEPHYR_EMUL_SENSOR_ICM42688_ICM42688_EMUL_H
```

Comment:
OMG that is a long macro name. Why do you have `emul/sensor/icm42688` in the path twice?